Safety/emergency-response-guide.rst

```diff
@@ -0,0 +1,173 @@
+
+.. Include one Primary Author and list of Contributors (comma separated) between the asterisks (*):
+.. |author| replace:: *Alysha Shugart*
+.. If there are no contributors, write "none" between the asterisks. Do not remove the substitution.
+.. |contributors| replace:: *Giovanni Corvetto*
+
+.. This is the label that can be used as for cross referencing this procedure.
+.. Recommended format is "Directory Name"-"Title Name"  -- Spaces should be replaced by hyphens.
+.. _Safety-emergency-response-guide:
+.. Each section should includes a label for cross referencing to a given area.
+.. Recommended format for all labels is "Title Name"-"Section Name" -- Spaces should be replaced by hyphens.
+.. To reference a label that isn't associated with an reST object such as a title or figure, you must include the link an explicit title using the syntax :ref:`link text <label-name>`.
+.. An error will alert you of identical labels during the build process.
+
+########################
+Emergency Response Guide
+########################
+
+This is a basic emergency response guide organized by the type of emergency. 
+This is not a comprehensive first responder training guide.
+This does not replace safety training required to be on the summit.
+It is recommended to contact the safety team to complete summit driver's training in addition to the required summit training. 
+
+.. note::
+    Observing specialists serve as the Nighttime Safety Officers if there is not a member of the safety team physically on the summit.
+    If they take the decision to close the telescope or evacuate the summit for inclement weather, that decision is not negotiable.
+    The priority of safety will always be given to staff first, equipment second. 
+
+.. _emergency-response-guide-medical-emergency:
+
+Medical Emergency
+^^^^^^^^^^^^^^^^^
+
+.. Preconditions
+
+Contact the PolyClinic Paramedic at extension 577
+
+.. note::
+    If no one answers the telephone, use any Rubin radio tuned to Channel 1. Put out the call:
+    
+    - "Atento paramedico, habla [your name]."
+    Wait for a response.
+    If there is no response on the radio, call 440 to the Mountain Assistant.
+
+Describe the emergency:
+
+- Type of Emergency
+
+    My collegue is unconscious.
+        - "Mi colega esta inconsciente."
+
+    My collegue is not breathing.
+        - "Mi colega no respira."
+
+    My colleague has fallen.
+        - "Mi colega se ha caído."
+
+    My colleague has fainted.
+        - "Mi colega se ha desmayado."
+
+    My colleague is bleeding profusely.
+        - "Mi colega esta sangrando mucho."
+
+- Your name
+
+    My name is [your name].
+        - "Me llamo [your name]."
+
+- Your location
+
+    We are in [AuxTel/Main Building] [floor 1, 2, 3, ..].
+        - "Estamos en el [telescopio auxilliar/edificio principal] de Rubin, en el piso [uno, dos, tres, ...]."
+
+- How many people are injured
```

Comment:
Add spanish like above?
"[uno, dos, tres] personas herido"  -- fix that spanish if i botched that...

Reply:
Fixed.